docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -30,37 +30,59 @@ Below are described the steps you need to make to migrate from v5 to v6.
   | `disableSelectionOnClick`  | `disableRowSelectionOnClick`  |
   | `disableMultipleSelection` | `disableMultipleRowSelection` |
 
+### Removed props
+
+- The `disableIgnoreModificationsIfProcessingProps` prop was removed and its behavior when `true` was incorporated as the default behavior.
+  The old behavior can be restored by using `apiRef.current.stopRowEditMode({ ignoreModifications: true })` or `apiRef.current.stopCellEditMode({ ignoreModifications: true })`.
+- The `onColumnVisibilityChange` prop was removed. Use `onColumnVisibilityModelChange` instead.
+
 ### State access
 
 - The `gridSelectionStateSelector` selector was renamed to `gridRowSelectionStateSelector`.
+- The `gridColumnsSelector` selector was renamed to `gridColumnsStateSelector`.
```

Comment:
We stopped exporting it in https://github.com/mui/mui-x/pull/6693, but the migration guide wasn't updated.
There's an open issue https://github.com/mui/mui-x/issues/6698 that we can address in this PR

```suggestion
- The `gridColumnsSelector` selector was removed. Use more specific grid columns selectors instead.
```